cookbooks/bcpc-hadoop/attributes/zabbix_web_monitoring.rb

```diff
@@ -0,0 +1,25 @@
+default[:bcpc][:hadoop][:zabbix][:web_monitoring][:scenarios] = [
+
+# graphite web ui has issue
+  { :name         => "graphite_web_ui_issue",
+    :url          => "https://10.0.100.5:8888",
+    :description  => "The graphite web ui is not accessible sometimes",
+    :trigger_item => "web.test.fail", # 0 = OK, 1 = Failure
+    :trigger_func => "max(10m)",
+    :trigger_cond => ">0",
+    :severity     => 2, # 1 to 5, 5 is the highest
+    :enabled      => true,
+  },
+
+# graphite web ui is not accessible at all
+  { :name         => "graphite_web_ui_down",
+    :url          => "https://10.0.100.5:8888",
+    :description  => "The graphite web ui is not accessible at all",
+    :trigger_item => "web.test.fail", # 0 = OK, 1 = Failure
+    :trigger_func => "min(10m)",
+    :trigger_cond => "=1",
```

Comment:
Does `=1` here mean it will only match exactly one time, or will this continue to match after the first one?